PastaGA.py

```diff
@@ -0,0 +1,354 @@
+__author__ = 'theep_000'
+
+import random
+import operator
+from collections import Counter
+import sys
+import argparse
+
+mutate_chance = 90
+base_pop = 100
+generation_max = 100
+
+poss_ingredients = ["tomato sauce","pesto","mozzarella","alfredo sauce", "pasta shells","chile sauce", "pasta bowties", "parmesan","meatballs","pasta tubes","shredded chicken","basil", "spinach","chives","shrimp","mushroom","macaroni","feta cheese","noodles","sausage","olives"]
+
+class pasta:
+    #format and functions of the pasta chromosome
+
+    __gene_1 = ""
+    __gene_2 = ""
+    __gene_3 = ""
+    __gene_4 = ""
+    fitvalue = 0
+
+    def __init__ (self, gene_1, gene_2, gene_3, gene_4):
+        #creates object from 4 genes
+
+        self.__gene_1 = gene_1
+        self.__gene_2 = gene_2
+        self.__gene_3 = gene_3
+        self.__gene_4 = gene_4
+
+    def list_ing(self):
+        #gives a list of the genes
+        return [self.__gene_1, self.__gene_2, self.__gene_3, self.__gene_4]
+
+    def set_fitvalue(self):
+        # Use to set fit value for each ingredient.
+        fit_val = 0
```

Comment:
We can do better than this if-else statement. 

It seems your fitness is just an addition of what each ingredient contributes to the score. 

So, instead of this, try this:

1. Create a class to represent a "Gene", in this case ingredient. 
2. In that class, have a variable that merely keeps track of the type of ingredient. 
3. In this class, create a dictionary that maps ingredient name to score

Then, in this function do something like:

score = 0
for curr in my_chromosome.genes:
        score += score_dict[curr.name]

Now, you don't have to use this large if else. 

The dict could look like:

score_dict = {"tomato_sauce":7, "pesto":5} ...